.changeset/healthy-kiwis-mix.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
If you rename this file to `.changeset/techdocs-word1-word2-word3` then we don't have to be blocked by a review from the `@backstage/maintainers`. :)